cobbler/actions/buildiso/netboot.py

```diff
@@ -458,60 +460,60 @@ def make_shorter(self, distname: str) -> str:
         self.distmap[distname] = str(self.distctr)
         return str(self.distctr)
 
-    def _generate_netboot_system(self, system, cfglines: List[str], exclude_dns: bool):
-        """
-        Generates the ISOLINUX cfg configuration for any systems included in the image.
-        :param system: The system which the configuration should be generated for.
-        :param cfglines: The already existing lines of the configuration.
-        :param exclude_dns: If DNS configuration should be excluded or not.
-        """
-        self.logger.info("processing system: %s", system.name)
-        profile = system.get_conceptual_parent()
-        dist = profile.get_conceptual_parent()
-        distname = self.make_shorter(dist.name)
-        self.copy_boot_files(dist, self.isolinuxdir, distname)
+    def _render_isolinux_entry(self, append_line, menu_name, kernel_path) -> str:
```

Comment:
Codacy says this signature is incompatible with supertype annotation. Is that on purpose and if yes why?

Reply:
It was on purpose, basically being a [partial function](https://docs.python.org/3/library/functools.html#functools.partial). But the same thing is achieved by just setting a default for the menu_indent value in the superclass and not passing it in this class. The idea was to not specify the menu_indent here because it does not matter (based on the existing code, I just assumed that this behavior is correct).

Reply:
Let's add a reasonable default and then omit it here since you mentioned that this is on the table. I see no downside of doing it this way and there is a reason (albeit not related to this scenario IMHO) why the linter has this rule.